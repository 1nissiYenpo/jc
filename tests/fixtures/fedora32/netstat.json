[{"proto": "tcp", "recv_q": 0, "send_q": 0, "local_address": "localhost.localdoma", "foreign_address": "192.168.71.1", "state": "ESTABLISHED", "kind": "network", "local_port": "ssh", "foreign_port": "52882", "transport_protocol": "tcp", "network_protocol": "ipv4", "foreign_port_num": 52882}, {"proto": "udp", "recv_q": 0, "send_q": 0, "local_address": "localhost.locald", "foreign_address": "192.168.71.254", "state": "ESTABLISHED", "kind": "network", "local_port": "bootpc", "foreign_port": "bootps", "transport_protocol": "udp", "network_protocol": "ipv4"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 36371, "path": "/run/user/1000/systemd/notify", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "DGRAM", "state": null, "inode": 15453, "path": "/run/systemd/notify", "kind": "socket"}, {"proto": "unix", "refcnt": 9, "flags": null, "type": "DGRAM", "state": null, "inode": 15471, "path": "/run/systemd/journal/dev-log", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 165956, "path": "@userdb-9bf8f59ca8f2bcce47a377edbaf8985d", "kind": "socket"}, {"proto": "unix", "refcnt": 12, "flags": null, "type": "DGRAM", "state": null, "inode": 15479, "path": "/run/systemd/journal/socket", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 29064, "path": "/var/run/chrony/chronyd.sock", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 165866, "path": "@userdb-3f208fa7c8c6d98822a696ee7ca5e3ad", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 165863, "path": "@userdb-da253950013f5ea3bc51a4049480d04e", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 165865, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 36476, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31389, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 30999, "path": "/var/lib/sss/pipes/private/sbus-dp_implicit_files.692", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 33770, "path": "/run/dbus/system_bus_socket", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 30731, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 24653, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31370, "path": "/var/lib/sss/pipes/private/sbus-dp_implicit_files.692", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 165862, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 33725, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 33885, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31173, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31019, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 33886, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 30735, "path": "/var/lib/sss/pipes/private/sbus-monitor", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 166231, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 30987, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 24745, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 165955, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 30575, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31062, "path": "/run/dbus/system_bus_socket", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 35419, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31399, "path": "/run/dbus/system_bus_socket", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 32430, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 29403, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 166232, "path": "/var/lib/sss/pipes/nss", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 31127, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 29538, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 33876, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31020, "path": "/var/lib/sss/pipes/private/sbus-monitor", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 30998, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 36475, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 36002, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 33769, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31390, "path": "/var/lib/sss/pipes/private/sbus-monitor", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 33132, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31398, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 30022, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "DGRAM", "state": null, "inode": 24748, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "DGRAM", "state": null, "inode": 24747, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 36348, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 32085, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31168, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "DGRAM", "state": null, "inode": 15455, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 30830, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 26911, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 35960, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 24662, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 36345, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 32538, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 28747, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31026, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "DGRAM", "state": null, "inode": 28615, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31713, "path": "/run/dbus/system_bus_socket", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31167, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 35989, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 36303, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31809, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 29396, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 34911, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "DGRAM", "state": null, "inode": 36374, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 32241, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 32057, "path": "/run/dbus/system_bus_socket", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 32000, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 33083, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31017, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "DGRAM", "state": null, "inode": 36373, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 34101, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 32539, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 26914, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 29395, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31022, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "DGRAM", "state": null, "inode": 15456, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 29251, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 26915, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 33052, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31999, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 29496, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 28400, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 36378, "path": "/run/dbus/system_bus_socket", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31808, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 29250, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 24582, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 32056, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31347, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 28678, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 30980, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31348, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 28677, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 34102, "path": "/run/dbus/system_bus_socket", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 24805, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 29322, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 36357, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 30832, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 26913, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 32010, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 28748, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 28602, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 28401, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31069, "path": "/run/dbus/system_bus_socket", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "DGRAM", "state": null, "inode": 28616, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 32334, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31712, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 36302, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 32333, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 32093, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 26912, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 35774, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 29321, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31025, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 28470, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 32094, "path": "/run/dbus/system_bus_socket", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31259, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "DGRAM", "state": null, "inode": 36471, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31054, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31064, "path": "/run/dbus/system_bus_socket", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 28471, "path": "/run/systemd/journal/stdout", "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 36377, "kind": "socket"}, {"proto": "unix", "refcnt": 2, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 35788, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31258, "kind": "socket"}, {"proto": "unix", "refcnt": 3, "flags": null, "type": "STREAM", "state": "CONNECTED", "inode": 31065, "path": "/run/dbus/system_bus_socket", "kind": "socket"}]
