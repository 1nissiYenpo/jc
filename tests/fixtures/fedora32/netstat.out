Active Internet connections (w/o servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State      
tcp        0      0 localhost.localdoma:ssh 192.168.71.1:52882      ESTABLISHED
udp        0      0 localhost.locald:bootpc 192.168.71.254:bootps   ESTABLISHED
Active UNIX domain sockets (w/o servers)
Proto RefCnt Flags       Type       State         I-Node   Path
unix  2      [ ]         DGRAM                    36371    /run/user/1000/systemd/notify
unix  3      [ ]         DGRAM                    15453    /run/systemd/notify
unix  9      [ ]         DGRAM                    15471    /run/systemd/journal/dev-log
unix  2      [ ]         DGRAM                    165956   @userdb-9bf8f59ca8f2bcce47a377edbaf8985d
unix  12     [ ]         DGRAM                    15479    /run/systemd/journal/socket
unix  2      [ ]         DGRAM                    29064    /var/run/chrony/chronyd.sock
unix  2      [ ]         DGRAM                    165866   @userdb-3f208fa7c8c6d98822a696ee7ca5e3ad
unix  2      [ ]         DGRAM                    165863   @userdb-da253950013f5ea3bc51a4049480d04e
unix  2      [ ]         DGRAM                    165865   
unix  3      [ ]         STREAM     CONNECTED     36476    
unix  3      [ ]         STREAM     CONNECTED     31389    
unix  3      [ ]         STREAM     CONNECTED     30999    /var/lib/sss/pipes/private/sbus-dp_implicit_files.692
unix  3      [ ]         STREAM     CONNECTED     33770    /run/dbus/system_bus_socket
unix  3      [ ]         STREAM     CONNECTED     30731    
unix  2      [ ]         DGRAM                    24653    
unix  3      [ ]         STREAM     CONNECTED     31370    /var/lib/sss/pipes/private/sbus-dp_implicit_files.692
unix  2      [ ]         DGRAM                    165862   
unix  2      [ ]         DGRAM                    33725    
unix  3      [ ]         STREAM     CONNECTED     33885    
unix  3      [ ]         STREAM     CONNECTED     31173    
unix  3      [ ]         STREAM     CONNECTED     31019    
unix  3      [ ]         STREAM     CONNECTED     33886    
unix  3      [ ]         STREAM     CONNECTED     30735    /var/lib/sss/pipes/private/sbus-monitor
unix  3      [ ]         STREAM     CONNECTED     166231   
unix  2      [ ]         DGRAM                    30987    
unix  2      [ ]         DGRAM                    24745    
unix  2      [ ]         DGRAM                    165955   
unix  2      [ ]         DGRAM                    30575    
unix  3      [ ]         STREAM     CONNECTED     31062    /run/dbus/system_bus_socket
unix  2      [ ]         STREAM     CONNECTED     35419    
unix  3      [ ]         STREAM     CONNECTED     31399    /run/dbus/system_bus_socket
unix  2      [ ]         DGRAM                    32430    
unix  2      [ ]         DGRAM                    29403    
unix  3      [ ]         STREAM     CONNECTED     166232   /var/lib/sss/pipes/nss
unix  2      [ ]         DGRAM                    31127    
unix  2      [ ]         DGRAM                    29538    
unix  2      [ ]         STREAM     CONNECTED     33876    
unix  3      [ ]         STREAM     CONNECTED     31020    /var/lib/sss/pipes/private/sbus-monitor
unix  3      [ ]         STREAM     CONNECTED     30998    
unix  3      [ ]         STREAM     CONNECTED     36475    
unix  2      [ ]         DGRAM                    36002    
unix  3      [ ]         STREAM     CONNECTED     33769    
unix  3      [ ]         STREAM     CONNECTED     31390    /var/lib/sss/pipes/private/sbus-monitor
unix  2      [ ]         STREAM     CONNECTED     33132    
unix  3      [ ]         STREAM     CONNECTED     31398    
unix  3      [ ]         STREAM     CONNECTED     30022    
unix  3      [ ]         DGRAM                    24748    
unix  3      [ ]         DGRAM                    24747    
unix  2      [ ]         DGRAM                    36348    
unix  2      [ ]         DGRAM                    32085    
unix  3      [ ]         STREAM     CONNECTED     31168    /run/systemd/journal/stdout
unix  3      [ ]         DGRAM                    15455    
unix  3      [ ]         STREAM     CONNECTED     30830    
unix  3      [ ]         STREAM     CONNECTED     26911    
unix  3      [ ]         STREAM     CONNECTED     35960    
unix  2      [ ]         DGRAM                    24662    
unix  2      [ ]         STREAM     CONNECTED     36345    
unix  3      [ ]         STREAM     CONNECTED     32538    
unix  3      [ ]         STREAM     CONNECTED     28747    
unix  3      [ ]         STREAM     CONNECTED     31026    
unix  3      [ ]         DGRAM                    28615    
unix  3      [ ]         STREAM     CONNECTED     31713    /run/dbus/system_bus_socket
unix  3      [ ]         STREAM     CONNECTED     31167    
unix  3      [ ]         STREAM     CONNECTED     35989    /run/systemd/journal/stdout
unix  3      [ ]         STREAM     CONNECTED     36303    /run/systemd/journal/stdout
unix  3      [ ]         STREAM     CONNECTED     31809    /run/systemd/journal/stdout
unix  3      [ ]         STREAM     CONNECTED     29396    /run/systemd/journal/stdout
unix  2      [ ]         STREAM     CONNECTED     34911    
unix  3      [ ]         DGRAM                    36374    
unix  2      [ ]         STREAM     CONNECTED     32241    
unix  3      [ ]         STREAM     CONNECTED     32057    /run/dbus/system_bus_socket
unix  3      [ ]         STREAM     CONNECTED     32000    /run/systemd/journal/stdout
unix  2      [ ]         DGRAM                    33083    
unix  3      [ ]         STREAM     CONNECTED     31017    
unix  3      [ ]         DGRAM                    36373    
unix  3      [ ]         STREAM     CONNECTED     34101    
unix  3      [ ]         STREAM     CONNECTED     32539    /run/systemd/journal/stdout
unix  3      [ ]         STREAM     CONNECTED     26914    
unix  3      [ ]         STREAM     CONNECTED     29395    
unix  3      [ ]         STREAM     CONNECTED     31022    
unix  3      [ ]         DGRAM                    15456    
unix  3      [ ]         STREAM     CONNECTED     29251    /run/systemd/journal/stdout
unix  3      [ ]         STREAM     CONNECTED     26915    
unix  2      [ ]         STREAM     CONNECTED     33052    
unix  3      [ ]         STREAM     CONNECTED     31999    
unix  2      [ ]         DGRAM                    29496    
unix  3      [ ]         STREAM     CONNECTED     28400    
unix  3      [ ]         STREAM     CONNECTED     36378    /run/dbus/system_bus_socket
unix  3      [ ]         STREAM     CONNECTED     31808    
unix  3      [ ]         STREAM     CONNECTED     29250    
unix  3      [ ]         STREAM     CONNECTED     24582    
unix  3      [ ]         STREAM     CONNECTED     32056    
unix  3      [ ]         STREAM     CONNECTED     31347    
unix  3      [ ]         STREAM     CONNECTED     28678    /run/systemd/journal/stdout
unix  2      [ ]         DGRAM                    30980    
unix  3      [ ]         STREAM     CONNECTED     31348    /run/systemd/journal/stdout
unix  3      [ ]         STREAM     CONNECTED     28677    
unix  3      [ ]         STREAM     CONNECTED     34102    /run/dbus/system_bus_socket
unix  3      [ ]         STREAM     CONNECTED     24805    /run/systemd/journal/stdout
unix  3      [ ]         STREAM     CONNECTED     29322    /run/systemd/journal/stdout
unix  2      [ ]         DGRAM                    36357    
unix  3      [ ]         STREAM     CONNECTED     30832    /run/systemd/journal/stdout
unix  2      [ ]         DGRAM                    26913    
unix  2      [ ]         STREAM     CONNECTED     32010    
unix  3      [ ]         STREAM     CONNECTED     28748    /run/systemd/journal/stdout
unix  2      [ ]         DGRAM                    28602    
unix  3      [ ]         STREAM     CONNECTED     28401    /run/systemd/journal/stdout
unix  3      [ ]         STREAM     CONNECTED     31069    /run/dbus/system_bus_socket
unix  3      [ ]         DGRAM                    28616    
unix  3      [ ]         STREAM     CONNECTED     32334    /run/systemd/journal/stdout
unix  3      [ ]         STREAM     CONNECTED     31712    
unix  3      [ ]         STREAM     CONNECTED     36302    
unix  3      [ ]         STREAM     CONNECTED     32333    
unix  3      [ ]         STREAM     CONNECTED     32093    
unix  3      [ ]         STREAM     CONNECTED     26912    
unix  2      [ ]         STREAM     CONNECTED     35774    
unix  3      [ ]         STREAM     CONNECTED     29321    
unix  3      [ ]         STREAM     CONNECTED     31025    
unix  3      [ ]         STREAM     CONNECTED     28470    
unix  3      [ ]         STREAM     CONNECTED     32094    /run/dbus/system_bus_socket
unix  3      [ ]         STREAM     CONNECTED     31259    /run/systemd/journal/stdout
unix  2      [ ]         DGRAM                    36471    
unix  3      [ ]         STREAM     CONNECTED     31054    
unix  3      [ ]         STREAM     CONNECTED     31064    /run/dbus/system_bus_socket
unix  3      [ ]         STREAM     CONNECTED     28471    /run/systemd/journal/stdout
unix  3      [ ]         STREAM     CONNECTED     36377    
unix  2      [ ]         STREAM     CONNECTED     35788    
unix  3      [ ]         STREAM     CONNECTED     31258    
unix  3      [ ]         STREAM     CONNECTED     31065    /run/dbus/system_bus_socket
Active Bluetooth connections (w/o servers)
Proto  Destination       Source            State         PSM DCID   SCID      IMTU    OMTU Security
Proto  Destination       Source            State     Channel

